src/containers/quiz/question-answer/Answer.styles.ts

```diff
@@ -0,0 +1,28 @@
+type AnswerState = 'correct' | 'incorrect'
+
+interface Params {
+  state?: AnswerState
+  isOpenAnswer?: boolean
+}
+
+export const styles = ({ state, isOpenAnswer }: Params) => {
+  const bgcolor = state && (state === 'correct' ? 'success.50' : 'error.50')
```

Comment:
Use enum here

Reply:
Thanks, used enum here and in other places!